thefuck/rules/git_clone_missing.py

```diff
@@ -0,0 +1,42 @@
+'''
+Rule: git_clone_missing
+
+Correct missing `git clone` command when pasting a git URL
+
+```sh
+& https://github.com/nvbn/thefuck.git
+git clone https://github.com/nvbn/thefuck.git
+
+Author: Miguel Guthridge
+'''
+from urllib import parse
+
+output_contains = [
+    'not found',
+    'no such file or directory',
+    'is not recognized as',
+]
+
+
+def match(command):
+    # Ignore capitalisation in output
+    output = command.output.lower()
+    script = command.script
+    # Check for a command not found error
+    if not any([search in output for search in output_contains]):
```

Comment:
Let's not create a list just to iterate over it:

```suggestion
    if not any(search in output for search in output_contains):
```